semgrep_output_v1.atd

```diff
@@ -255,6 +258,9 @@ type core_severity
   <python decorator="dataclass(frozen=True)"> = [
   | Error <json name="error">
   | Warning <json name="warning">
+  (* For the timeout/OOM during preprocessing errors, where it is not
```

Comment:
Not a fan, but if you think it's sufficiently enough different than Error, why not.

Reply:
I ended up killing it and just having a special case